db/warehouse/migrate/20241111212106_add_created_by_to_custom_assessment.rb

```diff
@@ -0,0 +1,14 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+class AddCreatedByToCustomAssessment < ActiveRecord::Migration[7.0]
+  def change
+    safety_assured do
+      # These refer to the users table in the app db (not warehouse), so fk relationship is not made explicitly
+      add_reference :CustomAssessments, :created_by_user, null: true, type: :integer
+      add_reference :CustomAssessments, :updated_by_user, null: true, type: :integer
```

Comment:
are you sure you want an integer here? Usually pks are `bigint` (which is the default)

```suggestion
      add_reference :CustomAssessments, :created_by_user, null: true
      add_reference :CustomAssessments, :updated_by_user, null: true
```

Reply:
I'm not sure, but I used `integer` because the pk of the `users` table is an `integer`:
https://github.com/greenriver/hmis-warehouse/blob/release-142/db/structure.sql#L3275

Maybe we intend to update this to `bigint`? Or maybe `users` is a table that we don't expect to ever grow that big?

Reply:
Thanks, my understanding is that we should avoid using integers for references. I think we use them now because many of the tables were migrated from sql server. I'd recommend using bigints and we can plan to transition the pks on those legacy tables in the future. Does that sound right @eanders ?

Reply:
If it allows `bigints` and the app works, then yes, the plan is to convert everything, so setting up now as `bigint` is one fewer data piece to migrate.

Reply:
Ok, perfect, thank you both!